pkgs/applications/misc/remarkable/remarkable-mouse/default.nix

```
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonApplication, fetchPypi, pythonPackages }:
```

make it more apparent that this is python3
```suggestion
{ stdenv, buildPythonApplication, fetchPypi, python3Packages }:
```